### DISCUSSION TITLE
Query regarding postprocessing

### DISCUSSION THREAD

**Discussion by Madhav-46** [xcompact3d/Incompact3d#326] [Q&A] [answered]
Hello Dr. Laizet and the Incompact3d Development Team,

I have been using Py4Incompact3d for postprocessing and am able to plot the mean quantities successfully. However, I would like to visualize the contours of these mean quantities in ParaView. Could you please guide me on the steps I need to take for this?

Specifically:

1.    Could it be achieved by exporting the mean quantities in a format compatible with ParaView (e.g., .vtk or .vtu files)? or

2.   Can we modify visu.f90 and variables.f90, so that we can save these mean quantities in .bin format and later be visualized through snapshot.xdmf

Any advice on how to achieve this, would be greatly appreciated!

Thank you for your support.

Best regards,
Madhava Rao

**Comment by slaizet** [comment, marked as answer]
Dear Madhava,
The mean quantities are binary files so it should not be a problem to look at them with Paraview. It should be easy to create a new xdmf file dedicated to the mean quantities.
Best,
Sylvain
